.gitmodules

```diff
@@ -1,6 +1,6 @@
 [submodule "gutenberg"]
 	path = gutenberg
-	url = ../../WordPress/gutenberg.git
+	url = https://github.com/callstack/gutenberg.git
```

Comment:
Should this stay with this value after the merge?

Reply:
No, it should be reverted before the merge